src/StochasticNoiseSimulator.cpp

```diff
@@ -321,97 +363,98 @@ void StochasticNoiseSimulator::applyNoiseOperation(const qc::Targets&
     }
 }
 
-dd::mEdge StochasticNoiseSimulator::generateNoiseOperation(bool                                    amplitudeDamping,
-                                                           dd::Qubit                               target,
-                                                           std::mt19937_64&                        engine,
-                                                           std::uniform_real_distribution<dd::fp>& distribution,
-                                                           dd::mEdge                               dd_operation,
-                                                           std::unique_ptr<dd::Package<>>&         localDD) {
-    dd::NoiseOperationKind effect;
-
-    for (const auto& noise_type: gate_noise_types) {
+dd::mEdge StochasticNoiseSimulator::generateNoiseOperation(const std::unique_ptr<StochasticNoiseSimulatorDDPackage>& localDD, dd::mEdge dd_operation,
```

Comment:
fits perfectly in the QFR where all the other operation DDs are constructed.

Reply:
The entire stochastic noise functionality is moved to the QFR